pkg/codegen/python/gen.go

```diff
@@ -1166,6 +1166,10 @@ func (mod *modContext) genResource(res *schema.Resource) (string, error) {
 		returnType := returnTypeObject(method.Function)
 		if returnType != nil {
 			mod.collectImportsForResource(returnType.Properties, imports, false /*input*/, res)
+		} else if t, ok := method.Function.ReturnsPlainType(); ok {
+			if rt, ok := t.(*schema.ResourceType); ok {
+				imports.addResource(mod, rt)
```

Comment:
Q: can this be an external type that needs imports as well, while not being a resource type?